packaging/build-release-zip.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
```

Comment:
I think we should add the command 'set -eu', so that the execution stops when there is an error. This is for cases where deleting the node modules fails and installing also fails, you will end up creating a bundle with the wrong modules

Reply:
Done in 05b96a1